firefox-ios/Client.xcodeproj/project.pbxproj

```diff
@@ -27944,7 +27963,7 @@
 					"-Xlinker",
 					"-no_application_extension",
 				);
-				SWIFT_TREAT_WARNINGS_AS_ERRORS = YES;
+				SWIFT_TREAT_WARNINGS_AS_ERRORS = NO;
```

Comment:
I am unsure why we see this change too here, since `SWIFT_TREAT_WARNINGS_AS_ERRORS` is `NO` on the main branch 🤔

Reply:
yeah my bad some commits crept in lol